src/assets/usage-examples-code/graphics/fill_triangle/fill_triangle.cpp

```diff
@@ -0,0 +1,15 @@
+#include "splashkit.h"
+
+int main()
+{
+    open_window("Fill triangle Example", 800, 600);
+    clear_screen();
+    fill_triangle(COLOR_RED, 100, 100, 200, 200, 300, 100);
+    refresh_screen();
+
+    delay(5000);
+
+    close_all_windows();
+
+    return 0;
```

Comment:
The code spacing here is a bit weird. My recommendation would be to group things around the `fill_triangle` function to emphasize it. E.g.

```c++
    open_window("Fill triangle Example", 800, 600);
    clear_screen();

    fill_triangle(COLOR_RED, 100, 100, 200, 200, 300, 100);
    refresh_screen();

    delay(5000);
    close_all_windows();

    return 0;
```